pkg/object/mqttproxy/spec.go

```diff
@@ -71,6 +71,9 @@ type (
 		ConnectionLimit      *RateLimit    `json:"connectionLimit" jsonschema:"omitempty"`
 		ClientPublishLimit   *RateLimit    `json:"clientPublishLimit" jsonschema:"omitempty"`
 		Rules                []*Rule       `json:"rules" jsonschema:"omitempty"`
+		BrokerMode           bool          `json:"brokerMode" jsonschema:"omitempty"`
```

Comment:
I prefer to enable the broker model by default  because:

1. backward compatible, the previous service can upgrade to the broker model by default instead of updating their spec.
2. Broker model has more extensive use cases.

Reply:
I am more prefer not to enable it because previous users can upgrade it and the behavior of MQTTProxy stays unchanged unless they update their spec... But I am open to update it.